DependencyInjection/Configuration.php

```diff
@@ -22,8 +22,14 @@ public function getConfigTreeBuilder()
 
         $rootNode
             ->children()
+                ->arrayNode('extra_fields')
```

Comment:
this fields are added in 'extra' not in 'context', i you prefer i can rename it "extra" ?